torch/lib/c10d/NCCLUtils.cpp

```diff
@@ -31,4 +31,62 @@ std::string ncclGetErrorWithVersion(ncclResult_t error) {
       getNcclVersion();
 }
 
+#if NCCL_VERSION_GE(2, 7, 0)
```

Comment:
@agolynski any reason these functions should be added to `NCCLUtils` instead of `ProcessGroupNCCL`? Do you plan to do this for all c10d APIs when working on #39662?